spec/lib/svg_sprite/svg_sprite_spec.rb

```diff
@@ -26,13 +26,12 @@
   end
 
   it "can search for a specific FA icon" do
-    expect(SvgSprite.search("fa-heart")).to match(/heart/)
```

Comment:
this would raise a deprecation warning, and once we take away the fa prefix, this test assertion becomes the same as the next line.